src/main/java/duke/commands/CommandHandler.java

```diff
@@ -0,0 +1,115 @@
+package duke.commands;
+
+import duke.exception.IllegalTaskCommandException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.task.*;

```

Comment:
Avoid importing using wildcard. Please import all classes explicitly. Check how you can disable the IDE from converting all imports to `*`